lms/envs/common.py

```diff
@@ -1328,6 +1328,20 @@
         ],
         'output_filename': 'css/lms-style-xmodule-annotations.css',
     },
+    'style-certificates': {
+        'source_filenames': [
+            'sass/certificates/main-ltr.css',
```

Comment:
@feanil @talbs was pre-compiling it and placing it there. However, to make sure that the SASS gets compiled, it is now placed within the sass folder. That places the compiled css next to the sass file, which is then pulled in by pipeline via this directive.